starters/features/auth/src/routes/plugin@auth.ts

```diff
@@ -1,15 +1,8 @@
-import { serverAuth$ } from "@builder.io/qwik-auth";
-import GitHub from "@auth/core/providers/github";
-import type { Provider } from "@auth/core/providers";
+import { QwikAuth$ } from "@auth/qwik";
```

Comment:
is it a requirement to use capital Q in `QwikAuth$` ? (because usually it's reserved for components no?)

Reply:
Yep, the library is implemented like that and it's using the same naming convention of the other auth.js integrations.

Reply:
Cool thanks